src/sentry/static/sentry/app/views/settings/project/projectKeys/list/index.tsx

```diff
@@ -28,8 +28,7 @@ import KeyRow from './keyRow';
 type Props = {
   organization: Organization;
   project: Project;
-} & RouteComponentProps<{orgId: string; projectId: string}, {}> &
-  AsyncView['props'];
+} & RouteComponentProps<{orgId: string; projectId: string}, {}>;
```

Comment:
Why does this not need the types from `AsyncView['props']`? Can we clean up `AsyncView['state']` too?

Reply:
@billyvg 
Since we are now adding the route props from this component, we don't need `AsyncView['props']` in our prop definition. However, we do need `AsyncView['state']` because we aren't providing the state fields in this particular component's state type.